doc/concepts/morphology.rst

```diff
@@ -176,18 +176,18 @@ tag 2 coloured grey for axon; tag 3 coloured light blue for basal dendrites.
   :width: 600
   :align: center
 
-  A ten segment cable cell, with soma (pink), axon (grey) and dendrite (light blue).
+  A ten-segment cable cell, with soma (pink), axon (grey), and dendrite (light blue).
   Python code to generate this cable cell is in the :class:`segment_tree<arbor.segment_tree>`
   documentation :ref:`here <morph-label-seg-code>`.
 
-* The tree is composed of 11 segments (1 soma, 2 axon, 8 dendrite).
-* The proximal ends of segments 0 and 9 (the soma and axon hillock respectively) are attached to the root of the tree.
+* The tree is composed of 11 segments (1 soma, 2 axons, 8 dendrites).
```

Comment:
Not sure the plural is best here. The line as written is trying to say:
We have 11 segments, thereof one labeled as the soma, 2 in the axon, and eight in the dendrite.